### ISSUE TITLE
Hide the waveform when playing recorded audio if `show_recording_waveform` is `False`

### ISSUE THREAD

**Issue by abidlabs** [gradio-app/gradio#6801]
Currently, if you set `show_recording_waveform` to `False` in `WaveformOptions` and pass it to `waveform_options` in `gr.Audio()`, it hides the waveform when recording audio, but still shows the waveform when playing the recorded audio. I think the expectation from users would to hide it when playing recorded audio as well.

**Reproduction**

```py
input_audio = gr.Audio(
    sources=["microphone"],
    waveform_options=gr.WaveformOptions(
        show_recording_waveform=False
    ),
)
demo = gr.Interface(
    fn=lambda x:x,
    inputs=input_audio,
    outputs="audio",
)

demo.launch()
```

cc @hannahblair

**Comment by abidlabs** [comment]
The previous trimming UI wasn't very good. I think bringing back the waveform view when trimming is okay. The main concern that the user had for disabling the waveform was the performance implications of rendering the waveform. So if the waveform is not shown by default, the performance concerns should be addressed?

**Comment by hboen** [comment]
i see that this issue is still not solved in the latest gradio-4.17.0. The waveformoption - False still not work.. Or there is another option to disable it?

**Comment by dwipper** [comment]
@abidlabs I'm on 4.21.0. The wave form stills shows when playing back a pre-recorded file. Is this the expected behavior?

`TtS_audio = gr.Audio(label="Audio", sources='upload', interactive=False, type='filepath', visible=True, format='mp3', autoplay=True, show_download_button=False, waveform_options=gr.WaveformOptions(show_recording_waveform=False))`

**Comment by abidlabs** [comment]
Yes currently that is the behavior @dwipper but this issue is exactly what you've pointed out, which is that we should also have an option (maybe the same option) to let developers disable the waveform for playback

**Comment by hboen** [comment]
it seems that this issue has NO high prio. I am at gradio 4.29 and it is still NOT solved. Why is this so difficult??? Make it default NO waveform (like in the early release)

**Comment by abidlabs** [comment]
> it seems that this issue has NO high prio. I am at gradio 4.29 and it is still NOT solved. Why is this so difficult??? Make it default NO waveform (like in the early release)

Hi @hboen, we understand your frustration, but please remember that this is an open-source repository maintained by a small team of contributors. Our team is working very hard in improving things but we do have competing priorities. If you have additional context for an issue or have a PR we can review, those are welcome but these kinds of "bumps" do not help. Feedback is always appreciated, but please  don't forget there are people at the other side of the screen please 🤗

**Comment by joshuacoffmanwwt** [comment]
Any status update on this? The waveforms are causing a memory leak. Every time the mic or audio player creates one the memory continues to bubble up and it would be nice if we could turn it off entirely.
<img width="784" alt="image" src="https://github.com/user-attachments/assets/40f34ba4-9eb9-4a12-b813-0afeb7d677fb">

**Comment by ROBERT-MCDOWELL** [comment]
any news of this issue? any options to hide everything but the play button?
